docker/backend.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /app
+
+COPY . /app
```

Comment:
I would advise against this because it'll prevent docker from using cache if any file changes in the source (so basically you're disabling cache all together). You should only copy what you actually need

Reply:
done